projects/swimlane/ngx-ui/src/lib/components/code-editor/code-editor.component.ts

```diff
@@ -71,23 +72,28 @@ const CODEMIRROR_VALUE_ACCESSOR = {
     './code-editor.component.scss'
   ]
 })
-export class CodeEditorComponent implements OnInit, AfterViewInit, OnDestroy, ControlValueAccessor {
-  @Input()
-  config: any = {
-    lineWrapping: true
-  };
+export class CodeEditorComponent implements OnInit, AfterViewInit, ControlValueAccessor {
+  @Input() config: any = { lineWrapping: true };
   @Input() theme: string = 'dracula';
   @Input() readOnly: any = false;
-  @Input() mode: any;
   @Input() autofocus: boolean = false;
-  @Input() lint: any;
   @Input() allowDropFileTypes: any[] = [];
-  @Input() lineNumbers: any;
   @Input() gutters: any[] = [];
+  @Input() mode?: any;
```

Comment:
do we know the values for mode? wonder if they're included in the codemirror types